content/events/2020-guadalajara/sponsor.md

```diff
@@ -3,64 +3,187 @@ Title = "Sponsor"
 Type = "event"
 Description = "Sponsor devopsdays Guadalajara 2020"
 +++
+<div class="container-fluid">
+  <div class="row justify-content-start">
+    <div class="col-md-9">
+      <div>
+      <h3></h3>
+      <p>We greatly value sponsors for this open event! If you are interested in sponsoring see a <a href="/events/2020-guadalajara/Prospectus.pdf">PDF of our prospectus</a> for more details or please drop us an email at {{< email_organizers >}}.</p>
+      <h1></h1>
+      <h1></h1>
```

Comment:
Do we need the empty headings?

Reply:
oks used "<br>" instead of empty heading